src/Features/Core/Portable/ImplementInterface/AbstractImplementInterfaceService.cs

```diff
@@ -88,6 +89,36 @@ private IEnumerable<CodeAction> GetActions(Document document, State state)
                     yield return ImplementInterfaceWithDisposePatternCodeAction.CreateImplementExplicitlyWithDisposePatternCodeAction(this, document, state);
                 }
             }
+
+            if (AnyImplementedImplicitly(state))
+            {
+                yield return ImplementInterfaceCodeAction.CreateImplementRemainingExplicitlyCodeAction(this, document, state);
+            }
+        }
+
+        private static bool AnyImplementedImplicitly(State state)
+        {
+            if (state.UnimplementedMembers.Length != state.UnimplementedExplicitMembers.Length)
+            {
+                return true;
+            }
+
+            for (var i = 0; i < state.UnimplementedMembers.Length; i++)
+            {
+                var (typeA, membersA) = state.UnimplementedMembers[i];
+                var (typeB, membersB) = state.UnimplementedExplicitMembers[i];
+                if (typeA != typeB)
```

Comment:
probably not the best idea to use `!=`.  Can you use `.Equals` for futureproofing this?